main_windows.go

```diff
@@ -30,11 +36,51 @@ var (
 
 	history      []string
 	historyIndex = 0
+
+	bookmarksFile string
+	bookmarks     []bookmark
+	bookmarksMenu *walk.Menu
 )
 
+type bookmark struct {
+	Address  string `json:"address"`
+	Password string `json:"password"`
```

Comment:
I agree, the assumption here was that most gameservers already configure rcon passwords in plaintext so it might not be such of a big risk. 

On Windows there's the [Wincred.h](https://docs.microsoft.com/en-us/windows/win32/api/wincred/) to store passwords but there might be a better way.